### ISSUE TITLE
question: why does it have to depend on webpack?

### ISSUE THREAD

**Issue by vyorkin** [TrySound/rollup-plugin-size-snapshot#47]
Could you please add a bit more info to the README about the implementation details, such as why it depends on webpack, etc?

**Comment by brody2consult** [comment]
Yeah I think it would be nice to have this dependency optional ... maybe with optionalPeerDependencies and a "webpack" option in the API?

Use of outdated Webpack v4 seems to cause issues with Node.js 17 that are resolved in recent Webpack v5 update ref:

- https://github.com/webpack/webpack/issues/14532#issuecomment-985083189
- https://github.com/microsoft/vscode-js-debug/pull/1146

related: I have similar thoughts for Parcel treeshake info from issue #17

It would be ideal if there could be a way to chain plugins to add Webpack and/or Parcel treeshake info.
